package/kernel/cifsd/Makefile

```diff
@@ -0,0 +1,57 @@
+include $(TOPDIR)/rules.mk
+include $(INCLUDE_DIR)/kernel.mk
+
+PKG_NAME:=cifsd
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/namjaejeon/cifsd.git
+PKG_SOURCE_DATE:=2019-02-15
+PKG_SOURCE_VERSION:=daf14f74fd105a338dc438beab042054e9720c43
+PKG_MIRROR_HASH:=521585ebfda0ecc02372b1a38ebf762fbbcead6d0b754a47599a5bf6bfdb3fb6
+
+PKG_MAINTAINER:=Andy Walsh <andy.walsh44+github@gmail.com>
+PKG_LICENSE:=GPL-2.0
+PKG_LICENSE_FILES:=COPYING
+
+include $(INCLUDE_DIR)/package.mk
+
+define KernelPackage/fs-cifsd
+	SUBMENU:=Filesystems
+	TITLE:=CIFS/SMB kernel server support
+	FILES:=$(PKG_BUILD_DIR)/cifsd.$(LINUX_KMOD_SUFFIX)
+	DEPENDS+= \
+		+kmod-nls-base \
+		+kmod-nls-utf8 \
+		+kmod-crypto-md4 \
+		+kmod-crypto-md5 \
+		+kmod-crypto-hmac \
+		+kmod-crypto-ecb \
+		+kmod-crypto-des \
+		+kmod-crypto-sha256 \
+		+kmod-crypto-cmac \
+		+kmod-crypto-sha512 \
+		+kmod-crypto-aead \
+		+kmod-crypto-ccm
+	KCONFIG:= \
+		CONFIG_KEYS=y \
```

Comment:
Why you add CONFIG_KEYS here it will be activated ijn all kernel images, please measure how much the size of a common kernel increases with this option set.

Reply:
It was a requirement for cifsd, so i added it here to ensure the kernel is compatible if this package is selected.

Reply:
has no impact on size

Reply:
Please make a PR/patch for both 4.14 and 4.19
I don't see any difference in size either on mvebu (arm)

Reply:
What you mean i already changed this in this PR for 4.14 and 4.19 ?
Should `CONFIG_KEYS=y` be its own PR?

Reply:
Perferably, at least a separate commit.

Reply:
done